.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: ruby
+rvm:
+  - 2.4.0
+
+script: 
+  - bundle exec bin/rake db:create 
+  - bundle exec bin/rake db:migrate 
+  - bundle exec rspec spec
+
+deploy:
+  provider: heroku
+  app: ace-book
+  api_key:
+    secure: "Zhq+bAklzTvNgYeVKtA5mi3GmTYqEzCWCGclgOuK20D23Yi1iPkbcH09ricLABlk1Y7qFB8jGtbplrPbGrQQdhUW9sQ9OGLRlA/vJnMhU4IVK/ZM1EJCFZPZPv5Zbg4s8L2r0Gz7uIEOX05TYFcREZ5iice4Mq89rX/SgOVPswh3KCFq0BfxaLq111WNAJtApH9amhuJbOUnRB7zMu38kpJcUqMsCFK3jeYBMbp4GxTyg+KK9MtiHS1tOrtXttx5EGaRyzBVi3JgFBo7gTaL3d03FJ6S5/3RFtDol3cUMSIfUVJkFlnzoedmyIgRq+PeDFIUZ17U6ZW8H09Rz9DjXBrpbzIGTEH92Xij7Fqd/BB2PE0ygSFwMcAxGvJHE5WJqNe6T5BhkATxGPXlYwRWMg3szZJdvlsu68xn6xAPdm2xH1DvdL1/ptcWltQFThjp3iy494sODbUInnjHPrj6zr1GsYlWnHigjcqi0IoJFjZRe55gM31cAadFYSBtm9wVoZHzyEje4MmjUwSUZMlaUVkv28eN5rdt8dbZYznu6IPUEY1nTFFipyiDq9WwidyziS4D6Flm5pbcNQBHyg+YwC1fzu3mcMSBnJtzqA60+IYao6iI+uI1eMX3+gSNdUVfEug8O6nuBxD5nCyojMlqmczai/s5oRJeTMT3PvITOV4="
```

Comment:
@ddosque this isn't the api-key, but the travis gem encrypted api key. we installed the travis gem and ran travis encrypt (api-key-hash) --add deploy.api_key to generate this change. we've redone this and also done this manually to the same result. we'll look into another way of doing this for heroku if we're off the mark.